mangle.json

```diff
@@ -37,7 +37,6 @@
       "$_args": "__H",
       "$_factory": "__h",
       "$_depth": "__b",
-      "$_nextDom": "__d",
```

Comment:
We should be able to remove post https://github.com/preactjs/preact/pull/4553, no?